build-image/first-boot.bash

```diff
@@ -64,7 +64,8 @@ if [[ -z "${username+x}" ]] || ! id $userdef &> /dev/null || id "$username" &> /
 else
   usermod -l "$username" "$userdef"
   usermod -m -d "/home/$username" "$username"
-  groupmod -n "$username" "$userdef"
+  groupmod -n "openhab" "$userdef"
```

Comment:
Doesn't the `openhab` group get created upon openHAB install?

Reply:
right it doesn't make sense that way

Reply:
But actually, it wont work correctly without it because at this point the openhab group has not been created. That is one of the reasons that Permissions Corrections was made, right? It runs after the openhab install and adds the user to the openhab group, right?

Reply:
IDK if permissions corrections was made for that reason and IDK where exactly openhab group is created but if you're right
then I would need to move changing the primary group into permissions_corrections(), right ?

Reply:
Probably, if I understand what is happening here correctly. @ThomDietrich any insight?

Reply:
moved it to after openhab install - wdyt?